tests/python/pytorch/graphbolt/test_base.py

```diff
@@ -13,17 +14,20 @@
 
 @unittest.skipIf(F._default_context_str == "cpu", "CopyTo needs GPU to test")
 def test_CopyTo():
-    item_sampler = gb.ItemSampler(gb.ItemSet(torch.randn(20)), 4)
+    def checkData(dp):
+        # Filtering DGLWarning:
+        #    Failed to map item list to `MiniBatch`
+        #    as the names of items are not provided.
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore", category=UserWarning)
+            for data in dp:
+                assert data.device.type == "cuda"
 
+    item_sampler = gb.ItemSampler(gb.ItemSet(torch.randn(20)), 4)
```

Comment:
I suggest to add `names` for `ItemSet` to eliminate warnings from the root instead of filtering.

Reply:
I am sorry but with that changes
```
 item_sampler = gb.ItemSampler(gb.ItemSet(torch.randn(20), names="seed_nodes"), 4)
```
the `test_CopyTo` fails:
```
=============================================================================================== FAILURES ================================================================================================
______________________________________________________________________________________________ test_CopyTo ______________________________________________________________________________________________

    @unittest.skipIf(F._default_context_str == "cpu", "CopyTo needs GPU to test")
    def test_CopyTo():
        item_sampler = gb.ItemSampler(gb.ItemSet(torch.randn(20), names="seed_nodes"), 4)
        #item_sampler = gb.ItemSampler(gb.ItemSet(torch.randn(20)), 4)
        import pdb; pdb.set_trace()
   
        # Invoke CopyTo via class constructor.
        dp = gb.CopyTo(item_sampler, "cuda")
        for data in dp:
>           assert data.device.type == "cuda"
E           AttributeError: 'MiniBatch' object has no attribute 'device'

tests/python/pytorch/graphbolt/test_base.py:21: AttributeError
```
This is what it looks like in `pdb`:
```
> /opt/dgl/dgl-source/tests/python/pytorch/graphbolt/test_base.py(21)test_CopyTo()
-> assert data.device.type == "cuda"
(Pdb) dir(data.device)
*** AttributeError: 'MiniBatch' object has no attribute 'device'
(Pdb)
```
Without adding `names` to  `ItemSet`
```
item_sampler = gb.ItemSampler(gb.ItemSet(torch.randn(20)), 4)
```
this test has no problems, and in the same line we can see::
```
> /opt/dgl/dgl-source/tests/python/pytorch/graphbolt/test_base.py(21)test_CopyTo()
-> assert data.device.type == "cuda"
(Pdb) dir(data.device)
['__class__', '__delattr__', '__dir__', '__doc__', '__enter__', '__eq__', '__exit__', '__format__', '__ge__', '__getattribute__', '__gt__', '__hash__', '__init__', '__init_subclass__', '__le__', '__lt__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__str__', '__subclasshook__', 'index', 'type']
(Pdb)
```